apps/codeforafrica/src/components/Page/Page.js

```diff
@@ -4,26 +4,35 @@ import React from "react";
 
 import Footer from "@/codeforafrica/components/Footer";
 
-function Page({ children, footer, title }) {
+function Page({ children, footer, menu, title }) {
```

Comment:
```suggestion
function Page({ children, footer, title }) {
```

`menu` (for footer) should come in `footer`.